### PR TITLE
Add note about scheduling retrospectives for failed deploys

### PR DESCRIPTION
**Pull request by zachmargolis**

### CODE REVIEW

File: _articles/appdev-deploy.md [hunk 1]
```diff
@@ -254,6 +254,13 @@ aws-vault exec prod-power -- ./bin/scale-remove-new-instances prod worker
 
 1. Recycle the app to get the new code out there (follow the [Production Deploy steps](#production))
 
+1. Schedule a [postmortem](#postmortem)
+
+##### Postmortem
+
+If you do end up rolling back a deploy, schedule a blameless postmortem afterwards. These help
```
Comment (zachmargolis) [inline _articles/appdev-deploy.md:261]: I was considering adding a note about who should be invited to these (code authors related to the reason for the rollback) but writing out out seemed obvious and maybe possible contradictory to the "blameless" bit

File: _articles/appdev-deploy.md [hunk 2]
```diff
@@ -254,6 +254,13 @@ aws-vault exec prod-power -- ./bin/scale-remove-new-instances prod worker
 
 1. Recycle the app to get the new code out there (follow the [Production Deploy steps](#production))
 
+1. Schedule a [postmortem](#postmortem)
```
Comment (aduth) [inline _articles/appdev-deploy.md:257]: Should we consider alternative language like "retrospective" ?

https://github.com/18F/charlie/blob/main/InclusionBot.md#trauma-triggering

**Review by aduth — APPROVED** [review]
Small suggestion on language, but otherwise LGTM 👍

File: _articles/appdev-deploy.md (hunk 2 above)
Comment (zachmargolis) [reply _articles/appdev-deploy.md]: good reminder, updated in 31c24f9054b3f5763d289255e43a70e97ad1c2e5

**Review by mitchellhenke — APPROVED** [review]

**Pull request merged**